src/components/header/Header.tsx

```diff
@@ -23,10 +25,10 @@ class Header extends React.Component<ConnectedHeaderProps> {
                 <WebsiteLogo width={250} height={80} />
                 <hr className="vertical-line" />
                 <div className="title-website">Payments platform</div>
-                {this.props.isLoginPage && <span className="alpha-sign">ALPHA</span>}
+                {this.props.loggedInStatus === LoginStatus.loggedOut && <span className="alpha-sign">ALPHA</span>}

```

Comment:
So the alpha sign will disappear as soon as you press the login button?
```suggestion
                {this.props.loggedInStatus !== LoginStatus.loggedIn && <span className="alpha-sign">ALPHA</span>}

```

Reply:
yeah, Catherine said so, well she actually said it should only appear on the login page so your suggestion is better